src/plugins/FetchStickerDataPlugin.ts

```diff
@@ -72,7 +72,10 @@ async function getAllStickerPacks(inputFile: string): Promise<Array<StickerPackP
         await fs.writeJson(cachePath, stickerPackPartial);
       }
 
-      stickerPackPartials.push(stickerPackPartial);
+      // Order partials in the reverse order of stickers.yml, so the most
+      // recently added pack is first (assuming stickers.yml is only ever
+      // appended to when new packs are added).
+      stickerPackPartials[stickerPackEntries.length - index - 1] = stickerPackPartial;
```

Comment:
I'd avoid changing this reference here. We may have up to 6 of these functions running at the same time, and is possible that two of them try to change the reference at the same time, resulting in dropped sticker packs.

`unshift` should get what we want here.

Reply:
`unshift` doesn't get us what we want. The fetches complete in somewhat random order since they are running in parallel, so our resulting `stickerData.json` was in a random order. We either need to assign them to the desired index (as I'm doing here) or re-sort after all the fetches have completed.

Reply:
I also don't see how this would ever result in a dropped sticker pack. Each pack is assigning to a unique index.

Reply:
I also don't see how this would ever result in a dropped sticker pack. Each pack is assigning to a unique index. e.g. if `index` was 12 in `stickers.yml` and we had 30 stickers, then this is assigning to index 17 (30 - 12 - 1) in the `stickerData.json`

Reply:
To be clear, what we're essentially doing is `stickerPackPartials[index] = stickerPackPartial;` and then reversing `stickerPackPartials` at the end, just in one step rather than two. Before what we were doing was just pushing the packs into `stickerPackPartials` as they completed in an indeterminate order. Unshifting would result in the same indeterminate order behavior.

Reply:
Mis-read the line, thanks for clarifying!